tests/test_e2e.py

```diff
@@ -0,0 +1,40 @@
+from datetime import datetime
+
+import neptune
+import tensorflow as tf
+
+from neptune_tensorboard import enable_tensorboard_logging
+
+
+def test_logging():
+    run = neptune.init_run()
```

Comment:
Use contextmanagers here like `with neptune.Run() as run:` as the standard `init_run()` without a `stop()` call will still have a background synchronization job that will be killed after the tests run and produce an ugly warning.